components/dot_menu/dot_menu.tsx

```diff
@@ -50,7 +50,6 @@ type Props = {
     channelIsArchived?: boolean; // TechDebt: Made non-mandatory while converting to typescript
     currentTeamUrl?: string; // TechDebt: Made non-mandatory while converting to typescript
     appBindings: AppBinding[];
-    appsEnabled: boolean;
```

Comment:
I have mixed feelings on removing this. The point of having this is to make sure no apps code is executed if the apps are not enabled. It is true that getAppBindings should be returning an empty list of bindings, but still, I prefer to have this completely covered. (0/5)

Reply:
@larkox I've made it so the `appsEnabled` value is back in the props. Do you think we should be passing the `isSystemMessage` value into the child, and have it use that as well? Or just keep it as is, using the value in the `index` file to determine if we pass on the bindings?

Reply:
So far, I think we only need that. The logic seems clear to me. If we ever need to make any distinction in the component about the system messages we can always reintroduce it.